src/app/events/dto.rs

```diff
@@ -22,6 +23,7 @@ impl From<crate::entity::event::EventEntity> for EventDto {
             end: event.end.to_string(),
             all_day: event.all_day,
             location: event.location,
+            original: event.original,

```

Comment:
Is original a good name for this? Wasn't it supposed to be an hash?

Reply:
Acoording to RFC5545 (*Internet Calendaring and Scheduling Core Object Specification*) an UID is mandatory to be present in every component of an iCalendar (see [page 117](https://www.rfc-editor.org/rfc/rfc5545) of the specifications). It it also required that the UID is globally unique.

> The "UID" itself **MUST** be a globally unique identifier.

Therefore we can assume that UIDs can be used to uniquely identify one iCalendar component, and that each UID is globally unique and does not interfere with any other UID from another iCal source.

Reply:
In that case, it is just about a different name. In my opinion `original` does not describe it well. `original_identifier ` like you proposed is a possibility, or something along the lines of `event_uid`, since our events will need them too for exporting.

Reply:
Oh btw something that I just realize, do we want to send the that UID back through the API? Because it has been added to the DTO

Reply:
Renamed to `event_uid` and removed from dto.